spec/requests/rack_attack_spec.rb

```diff
@@ -25,6 +25,30 @@ def unique_user_params
     expect(response).to have_http_status(:redirect)
   end
 
+  it "successful response does not include retry-after header" do
+    get root_path, env: { "REMOTE_ADDR" => Faker::Internet.unique.ip_v4_address }
```

Comment:
Can you use `public_ip_v4_address` instead? `ip_v4_address` sometimes return an internal address which is not rate limited because of the `safelist_ip` rule, causing intermittent test failures. (Please feel free to update the existing uses of `ip_v4_address` in this file as well.